src/components/media-item/index.jsx

```diff
@@ -1,12 +1,22 @@
 import PropTypes from "prop-types";
 
-const COMPONENT_CLASS_NAME = "c-caption";
+const COMPONENT_CLASS_NAME = "c-media-item";
 
-const Caption = ({ caption, className, credit, title }) =>
-	title || caption || credit ? (
+const MediaItem = ({ alt, caption, className, credit, title, height, loading, src, width }) =>
+	title || caption || credit || src ? (
 		<figcaption
 			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
 		>
+			{src ? (
```

Comment:
We don't want to be duplicating logic of other "media" type components - the `<MediaItem>` component should accept any "media" and place it in the correct location in this component

Reply:
-  with not "duplicating logic", are you saying this should take in the `<Image>` component as children? I know in other component, we chose not to use the `<Link>` component for the `<Overline>`, right? 

in my questions area: 

> Just to clarify, we won't be nesting the Image component within this MediaItem component like
> const Feature = () => (
> <MediaItem>
> <Image>
> </MediaItem>
> )
> Instead, we want to duplicate the props and options necessary for showing an image within the MediaItem, right?


- ok. so any media, like video? any other, audio? 
- when you say, "place it in the correct location," do you mean that the `<img>` tag could be below the caption? I'm not sure I follow 

@matthewroach

Reply:
cool makes sense